build.psm1

```diff
@@ -271,7 +271,7 @@ function Start-PSBuild {
         Push-Location $PSScriptRoot
         try {
             # Excluded sqlite3 folder is due to this Roslyn issue: https://github.com/dotnet/roslyn/issues/23060
-            git clean -fdx --exclude .vs/PowerShell/v15/Server/sqlite3
+            git clean -fdx --exclude .vs/PowerShell/v15/Server/sqlite3 --exclude src/Modules/nuget.config
```

Comment:
A comment above this explains why they excluded `sqlite3`.
For consistency, it would be good to add a similar comment about `--exclude src/Modules/nuget.config`.